pkgs/by-name/fr/freej2me/package.nix

```
@@ -0,0 +1,87 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  ant,
+  jdk,
+  stripJavaArchivesHook,
+  writeShellScriptBin,
+}:
+
+stdenv.mkDerivation {
+  pname = "freej2me";
+  version = "0-unstable-2024-02-16";
+
+  src = fetchFromGitHub {
+    owner = "hex007";
+    repo = "freej2me";
+    rev = "8b9bc8a19baf26e3d92f88934a64a32f1cbc2795";
+    hash = "sha256-FaB2qnbtJnSZbrGRYDiz3Q/uNgD4GX//VaID6jHtyeU=";
+  };
+
+  nativeBuildInputs = [
+    ant
+    jdk
+    stripJavaArchivesHook
+  ];
+
+  prePatch = ''
+    substituteInPlace src/org/recompile/freej2me/Anbu.java \
+      --replace-fail "/usr/local" "${placeholder "out"}"
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+
+    ant
+    make -C src/libretro
+
+    runHook postBuild
+  '';
+
+  preInstall = ''
+    substituteInPlace src/sdl2/Makefile \
+      --replace-fail "/usr/local" "${placeholder "out"}"
+  '';
+
+  installPhase = let
+    mk-script-bin = frondend: writeShellScriptBin frondend ''
+      if [ ''$# -ne 3 ]; then
+        echo "usage: ${frondend} [midlet] [width] [height]"
```

This is no longer using `midlet` right?
```suggestion
        echo "usage: ${frondend} [width] [height]"
```

---

midlet is the java file provided to the script